pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -312,3 +312,8 @@ type ClusterWorkspaceShardList struct {
 
 	Items []ClusterWorkspaceShard `json:"items"`
 }
+
+const (
+	ClusterWorkspacePhaseLabel             = "clusterworkspace.tenancy.kcp.dev/phase"
```

Comment:
Add godoc for these?

Also, since we're using these on ClusterWorkspaces, would it possibly make sense to remove `clusterworkspace` from the keys?

Reply:
Might be nice to keep them just to make the thing globally unique for now.

Reply:
please use `*.internal.kcp.dev` for all the labels and annotation we consider internal and subject to change at any time.